pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml

```diff
@@ -668,11 +715,30 @@ spec:
                       - do-not-release
                       - released
                       type: string
-                    description: 'ReleaseIPs tracks the state for every IP considered
-                      for release. value can be one of the following string : * marked-for-release
-                      : Set by operator as possible candidate for IP * ready-for-release  :
-                      Acknowledged as safe to release by agent * do-not-release     :
-                      IP already in use / not owned by the node. Set by agent * released           :
+                    description: 'ReleaseIPs tracks the state for every IPv4 address
+                      considered for release. The value can be one of the following
+                      strings: * marked-for-release : Set by operator as possible
+                      candidate for IP * ready-for-release  : Acknowledged as safe
+                      to release by agent * do-not-release     : IP already in use
+                      / not owned by the node. Set by agent * released           :
+                      IP successfully released. Set by operator'
+                    type: object
+                  release-ipv6s:
+                    additionalProperties:
+                      description: IPReleaseStatus  defines the valid states in IP
```

Comment:
nit: extra whitespace in this (and original).

Reply:
@tommyp1ckles thanks for the review. I resolved ^ in commit 2eef811.